charts/generic/node-poweroff/experiment.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: litmuschaos.io/v1alpha1
+description:
+  message: |
+    poweroff node
+kind: ChaosExperiment
+metadata:
+  name: node-poweroff
+  version: 0.1.0
+  labels:
+    name: node-poweroff
+    app.kubernetes.io/part-of: litmus
+    app.kubernetes.io/component: chaosexperiment
+    app.kubernetes.io/version: latest
+spec:
+  definition:
+    scope: Cluster
+    permissions:
+      - apiGroups:
+          - ""
+          - "batch"
+          - "apps"
+          - "litmuschaos.io"
+        resources:
+          - "jobs"
+          - "pods"
+          - "pods/log"
+          - "events"
+          - "chaosengines"
+          - "chaosexperiments"
+          - "chaosresults"
+          - "secrets"
+        verbs:
+          - "create"
+          - "list"
+          - "get"
+          - "patch"
+          - "update"
+          - "delete"
+      - apiGroups:
+          - ""
+        resources:
+          - "nodes"
+        verbs:
+          - "get"
+          - "list"
+    image: "litmuschaos/go-runner:latest"
+    imagePullPolicy: Always
+    args:
+    - -c
+    - ./experiments -name node-restart
+    command:
+    - /bin/bash
+    env:
+    - name: SSH_USER
+      value: 'root'
+
+    - name: TOTAL_CHAOS_DURATION
+      value: '360'
+
+    - name: EXPERIMENT_NAME
+      value: 'node-poweroff'
+
+    - name: REBOOT_COMMAND
+      value: '-o ServerAliveInterval=1 -o ServerAliveCountMax=1 "sudo systemctl poweroff --force --force" ; true'
```

Comment:
There are two force flags here. Is it intended ?

Reply:
Yes. According to the `systemctl` [man page](https://www.systutorials.com/docs/linux/man/8-poweroff/):

-f, --force

Force immediate halt, power-off, or reboot. When specified once, this results in an immediate but clean shutdown by the system manager. **When specified twice**, this results in an immediate shutdown without contacting the system manager.

Reply:
The additional `-o ServerAliveInterval=1 -o ServerAliveCountMax=1` flags are used to force a quick disconnect from the server, otherwise the connection lingers on for a long time (5-6 minutes in my tests). The last command `true` is to avoid the command execution from returning an error code due to the forced disconnect. With an error code, the helper job reports as `Error`.

Reply:
Got it, thanks @jordigilh